portal/src/examples/MessageBoxExample.tsx

```diff
@@ -7,16 +7,24 @@ import messageType from "!raw-loader!@fremtind/jkl-message-box-react/build/Messa
 
 const example = `<>
         <InfoMessage title="Info">
-            Cupidatat Lorem incididunt incididunt in non mollit cillum Lorem eiusmod sunt magna.
+            Vi lagrer alle svarene dine underveis, slik at du kan forstsette der du slapp når det passer for deg.
         </InfoMessage>
         <ErrorMessage title="Error">
-            In non dolore ullamco minim adipisicing ipsum pariatur deserunt ipsum.
+            Vi finner ikke denne kombinasjonen av e-post og passord. Prøv igjen.
         </ErrorMessage>
-        <SuccessMessage title="Success">Ullamco minim aute Lorem adipisicing.</SuccessMessage>
-        <WarningMessage title="Warning">Culpa commodo velit cupidatat anim excepteur est culpa.</WarningMessage>
-        <InfoMessage fullWidth title="Full width info">Reprehenderit magna duis aliquip adipisicing do mollit ullamco. Pariatur quis ullamco eu cillum. Exercitation in mollit amet ex aute culpa do incididunt Lorem et non id quis ex. Pariatur ex cillum anim consectetur eu mollit occaecat ut aliqua mollit magna. Irure ut occaecat aliqua excepteur consectetur aliqua eiusmod qui excepteur consectetur magna aliqua.
-        </InfoMessage>
-</>`;
+        <SuccessMessage title="Success">Saken din er registrert</SuccessMessage>
+        <WarningMessage title="Warning">Du må fylle ut alle feltene før du kan gå videre</WarningMessage>
+        <InfoMessage fullWidth title="Full width info">
+## Slik behandler vi saken din
```

Comment:
@Gorild Var tanken at dette skulle være formatert text? Altså H3 + punktliste?

Det er ikke støtte for Markdown-formatering av kode, men jeg kan godt skrive om til å bruke H3 + punktliste.

Reply:
Det kan du gjøre

Reply:
Hei! Skrev til deg på Slack nettopp - får du til å vise meg hvordan dett blir seende ut "live"? Så kan jeg vurdere/revurdere.

Reply:
Da har jeg redigert i koden og commited til Patch-3. Se om du får det opp.